server/backend/api/app/crud/crud_user_terms_conditions.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from sqlalchemy.orm import Session
+
+from api.app.models.model import FamUserTermsConditions
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def if_needs_accept_terms_and_conditions(
+    db: Session, user_id: int, version: str
+) -> bool:
+    return (
+        False
+        if db.query(FamUserTermsConditions)
+        .filter(
+            FamUserTermsConditions.user_id == user_id,
+            FamUserTermsConditions.version == version,
+        )
+        .one_or_none()
```

Comment:
if found one, it will return "False", means user has agreed previously

if found none, then it will return "True", means user needs to agree

this logic is doing what we want here?

Reply:
Ah ha, I see what I was confused at.
I suggest in this case don't use compact form of the python condition statement with query, it looks confusing.